### ISSUE TITLE
Passing :namespaces is broken

### ISSUE THREAD

**Issue by martinklepsch** [crisptrutski/boot-cljs-test#17]
```
boot test-cljs -n org.martinklepsch.cc-set-test
Writing suite.cljs...
Writing output.cljs.edn...
Compiling ClojureScript...
• output.js
Running cljs tests...
```

And then it's stuck :smile:

**Comment by martinklepsch** [comment]
I briefly looked at the source to see if I can figure out what's going on but didn't have any helpful epiphanies unfortunately. This is my project: https://github.com/martinklepsch/custom-comparator-set. Might also be a nice example of testing clj & cljs further down the line.

**Comment by crisptrutski** [comment]
Thanks for the report, after poking around I found:
1. `doo-tests` is variadic, doesn't take the collection
2. `doo-tests` expects quoted symbols
3. your hadn't added tests to the `source-paths` 

Responses:
1. Will fix
2. I could put some magic around for hygiene (think that I should?) 
3. :stuck_out_tongue:

**Comment by martinklepsch** [comment]
1. :+1: 
2. Shouldn't `n namespaces NS  #{sym} "..."` mean that passed things should always be coerced to symbols?
3. Uhm... 😂 I must have had this in a top-level `set-env!` call. I tried the different commandline invocations and one definitely ran tests. Could be it didn't ran my tests though maybe I just saw some test output without explicitly checking that my own tests have been run. Oops 😄

Cheers, hope you're well! :)
